docs/walkthroughs/provision-keycloak-apb.adoc

```diff
@@ -0,0 +1,96 @@
+[[provision-keycloak-service]]
+= Provision KeyCloak Service
+
+== Provision
+Choose the type of service you would like to provision:
+
+* Keycloak service
+* Linked Keycloak service
+
+Go to the OpenShift web console. Open the *Mobile* Tab > *Services* from the service catalog and select the Keycloak service.
+This should open the Keycloak service provision wizard. Press the *Next* button and provide the configuration for the KeyCloak service
+
+=== KeyCloak service
+This service type can be shared between different Keycloak services.
+
+*The configuration would ask for the following parameters:*
```

Comment:
The configuration would ask for the following parameters > The configuration will ask for the following parameters